README.md

```diff
@@ -36,9 +36,8 @@ This is a budget app that allows users to track their expenses and incomes. User
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🚀 Live Demo <a name="live-demo"></a>
-
-- ✅ Click [here](https://budget-app-zipl.onrender.com) to see the project
-- ✅ Click [here](https://drive.google.com/file/d/1gBibLP7Bis7F4Y52T1bZ00zUceLGL6ti/view?usp=sharing) to see the video presentation.
+- ✅ Click [here](https://budget-application-2kkv.onrender.com) to see the project
```

Comment:
Great job on addressing the previous feedback! Just one small suggestion:

- [ ] Currently, there is a server-side error occurring in your project. From my experience, it seems there might be an issue with the database migration on the deployed link. Because most of the students forget to include data base migration commands. So, I recommend checking the remote database to ensure that the necessary tables exist. You can follow the [official documentation](https://docs.render.com/deploy-rails) for your chosen deployment method to troubleshoot and complete the migration process.

There are several ways to solve it. Here's a potential solution:
1. Go to the Render dashboard.
2. Navigate to your project.
3. In the bottom section called "Settings," find "Build & Deploy."
![image](https://github.com/evansnyamekye/budget-app/assets/55840999/de111149-01b8-4a75-86ec-364603382f87)
5. Look for an editable field named `Start Command.`
![image](https://github.com/evansnyamekye/budget-app/assets/55840999/d047c5e6-ba52-4024-90e5-cfc4fff4db96)
7. You may see a command like `bundle exec rails s`. I recommend changing it to `bundle exec rails db:migrate && bundle exec rails s`.
![image](https://github.com/evansnyamekye/budget-app/assets/55840999/6ad075e9-6cb1-4366-a829-2a05b9418e91)

I hope this resolves the issue. Alternatively, you can follow the official documentation, as there are several ways to approach this. Keep up the good work! 🚀👍